server/redis.js

```diff
@@ -1,8 +1,19 @@
+const { promisify } = require('util');
 const redis = require('redis');
 
+const cacheDurationInSeconds = 60 * 60;
 const REDIS_PORT = process.env.REDIS_PORT || 6379;
 
 const client = redis.createClient(REDIS_PORT);
-const cacheDuration = 1000 * 60 * 2;
 
-module.exports = { client, cacheDuration };
+const getAsync = promisify(client.get).bind(client);
+const setExAsync = promisify(client.setex).bind(client);
+const existsAsync = promisify(client.exists).bind(client);
+
```

Comment:
Is this necessary? promisify for the client server functions? I understand the need based on us making requests to the redis server, I guess I'm just a little surprised that the get, set, and exists methods are not inherently async from the redis lib.